src/printer.js

```diff
@@ -1729,6 +1729,31 @@ function genericPrintNoParens(path, options, print, args) {
     case "TemplateElement":
       return join(literalline, n.value.raw.split(/\r?\n/g));
     case "TemplateLiteral": {
+      /* Subtree parsing PoC */
+      const parent = path.getParentNode();
+      const parentParent = path.getParentNode(1);
+      const isStyledJsx =
+        n.quasis &&
+        n.quasis.length === 1 &&
+        parent.type === "JSXExpressionContainer" &&
+        parentParent.type === "JSXElement" &&
+        parentParent.openingElement.attributes.some(
+          attribute => attribute.name.name === "jsx"
+        );
+
+      if (isStyledJsx) {
+        const parseCss = require("./parser-postcss");
```

Comment:
You need to replace `require` by `eval("require")` otherwise it'll break code splitting and include the css parser in the main file.